src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -166,23 +150,22 @@ trait EntityComponent {
         _ <- deleteEntityAttributes(preExistingEntityRecs)
         savingEntityRecs <- insertNewEntities(workspaceContext, entities, preExistingEntityRecs).map(_ ++ preExistingEntityRecs)
         referencedAndSavingEntityRecs <- lookupNotYetLoadedReferences(workspaceContext, entities, savingEntityRecs).map(_ ++ savingEntityRecs)
-        _ <- insertAttributes(entities, referencedAndSavingEntityRecs)
+        _ <- insertAttributes(workspaceContext, entities, referencedAndSavingEntityRecs)
       } yield entities
     }
 
-    private def insertAttributes(entities: Traversable[Entity], entityRecs: Traversable[EntityRecord]) = {
+    private def insertAttributes(workspaceContext: SlickWorkspaceContext, entities: Traversable[Entity], entityRecs: Traversable[EntityRecord]) = {
```

Comment:
ah, it used to be used in an older commit of this branch but I forgot to remove it